### ISSUE TITLE
Bootstrap style applied to the color picker arrow

### ISSUE THREAD

**Issue by thomas-ama** [zefoy/ngx-color-picker#159]
I have a question about CSS isolation of the color picker.

I use it in a Bootstrap popover. Those popovers contain an arrow, and style is applied using the `arrow` class.

However, `ngx-color-picker` uses the same class and both bootstrap and `ngx-color-picker` are applied to the color picker, as shown in the attached screenshot.

<img width="248" alt="screenshot 2019-01-22 at 10 25 24" src="https://user-images.githubusercontent.com/46318805/51525660-1a995d00-1e31-11e9-862d-0585cb2a3a25.png">
My question is: how to prevent the color picker from inheriting the global style (bootstrap)?

Thanks.

**Comment by sconix** [comment]
I do not think you can so you just need to override the styles for the color picker.

**Comment by thomas-ama** [comment]
Thanks, I had to use global styles to override it due to view encapsulation (not the better way...).
